spec/graphql/query/executor_spec.rb

```diff
@@ -132,6 +132,22 @@
       end
     end
 
+    describe 'if nil is given for a non-null field' do
+      let(:query_string) {%| query noMilk { cantBeNullButIs }|}
+      it 'turns into error messages' do
+        expected = {
+          "data" => { "cantBeNullButIs" => nil },
```

Comment:
This field shouldn't be present in the response according to the graphql spec: http://facebook.github.io/graphql/#sec-Nullability